### PR TITLE
EIP-15: Updating docs

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://issues.openmrs.org/browse/EIP-15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# EIP application
+# EIP client
```
Comment (mks-d) [inline README.md]: Title Case please.

File: README.md [hunk 2]
```diff
@@ -44,5 +44,5 @@ Make sure the build completed successfully.
 Following the above file structure, copy the generated `.jar` file located at `eip-client/app/target/` path into the working root directory and run the app on commandline with the following command.
 
 ```
-java -jar eip-client-app-1.0.0-SNAPSHOT.jar
+java -jar client-app-1.0.0-SNAPSHOT.jar
```
Comment (mks-d) [inline README.md]: @rbuisson don't you think that the JAR itself should be named like `openmrs-eip-client-<version>.jar`?

**Review by mks-d — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Ruhanga) [reply README.md]: @rbuisson, simply pinging you here....

Comment (mks-d) [reply README.md]: @Ruhanga I think you can go ahead with my suggestion.

Comment (Ruhanga) [reply README.md]: Ok, thanks.

**Pull request merged**